sentry/src/main/java/io/sentry/Hub.java

```diff
@@ -700,12 +700,21 @@ public void flush(long timeoutMillis) {
   public @NotNull ITransaction startTransaction(
       final @NotNull TransactionContext transactionContext,
       final @NotNull TransactionOptions transactionOptions) {
-    return createTransaction(transactionContext, transactionOptions);
+    return startTransaction(transactionContext, transactionOptions, false);
+  }
+
+  @Override
+  public @NotNull ITransaction startTransaction(
+      final @NotNull TransactionContext transactionContext,
+      final @NotNull TransactionOptions transactionOptions,
+      final boolean isStartupTransaction) {
+    return createTransaction(transactionContext, transactionOptions, isStartupTransaction);
   }
 
   private @NotNull ITransaction createTransaction(
       final @NotNull TransactionContext transactionContext,
-      final @NotNull TransactionOptions transactionOptions) {
+      final @NotNull TransactionOptions transactionOptions,
+      final boolean isStartupTransaction) {
```

Comment:
`m`: can we put this boolean inside `TransactionOptions`? Ideally we'd keep the number of `startTransaction` overloads at a minimum whenever possible

Reply:
We can, the only thing is that `TransactionOption` is public and the we'd have to add the get method to `ITransaction`.
If we are fine with that, it's totally doable

Reply:
why would we need a new method in ITransaction?

Reply:
you're right, we don't. It's just late 😅